src/errors.rs

```diff
@@ -3,7 +3,7 @@ use core::fmt::{Display, Formatter, Result};
 use derive_more::Error;
 
 #[cfg_attr(feature = "std", derive(Error))]
-#[derive(Debug, PartialEq)]
+#[derive(Copy, Clone, Debug, PartialEq)]
```

Comment:
I'm convinced `Copy` instance should be removed

Reply:
removed